### PR TITLE
Update Backend Readme

### PR DESCRIPTION
**Pull request by mkrumholz**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Rubocop has passed locally and any fixes were made for failures
- [ ] CircleCI build is passing
- [ ] All tests passing locally

## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: 
closes #

## What is the new behavior?

- 
- 
- 

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,87 @@
...
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<img width="844" alt="distance_calculations" src="https://user-images.githubusercontent.com/26797256/128450433-1f90c288-b48e-4174-9f6b-ff0c1feb12b0.png">
 
-* ...
+<img width="670" alt="weighted_ratings" src="https://user-images.githubusercontent.com/26797256/128450437-8794b1c0-1c6e-4869-ab72-0bf9217275d1.png">
+
+As the user base grows in our app (and thus our database of user ratings also grows), the recommendation model will become more accurate in providing a recommendation to a particular user that the they will enjoy.
```
Comment (redferret) [inline README.md:36]: just a minor grammatical error `recommendation to a particular user that the they will enjoy.`
edit to `recommendation to a particular user that they will enjoy.`

**Review by redferret — APPROVED** [review]
Awesome read me 👍

**Review by twarbelow — APPROVED** [review]
I made a couple changes
- added links for DataFrame (since it is a key thing that Rails peeps from our cohort probably wouldn't know) and metrics (again, key thing that Rails peeps wouldn't necessarily know)
- very slight editing for clarity

**Review by Jaybraum — APPROVED** [review]

**Review by redferret — APPROVED** [review]

**Review by markcyen — APPROVED** [review]
Great catch

**Pull request merged**